docs/source/duckdb_cli.md

```diff
@@ -0,0 +1,57 @@
+# DuckDB CLI
+
+The [DuckDB CLI](https://duckdb.org/docs/api/cli/overview.html) (Command Line Interface) is a single, dependency-free executable. 
+
+<Tip>
+
+For installation details, visit the [installation page](https://duckdb.org/docs/installation).
+
+</Tip>
+
+Starting from version `v0.10.3`, the DuckDB CLI includes native support for accessing datasets on Hugging Face via URLs. Here are some features you can leverage with this powerful tool:
+
+- Query public, gated and private datasets
```

Comment:
This makes it sound like you can query other users/orgs gated and private datasets? Maybe something like:

```suggestion
- Query public datasets and your own gated and private datasets
```